lively.morphic/rendering/animations.js

```diff
@@ -125,11 +122,8 @@ export class PropertyAnimation {
       const shadowBefore = morph.dropShadow || new ShadowObject({ blur: 0, distance: 0, spread: 0 });
       const shadowAfter = config.dropShadow;
       config.customTween = fun.compose(p => {
-        morph.withMetaDo({ metaInteraction: true }, () => {
```

Comment:
What is the rule for when we need `metaInteraction` to be `true` and when not? Below it has been added multiple times.

Reply:
Originally this was a flag that was supposed to tell any tool that tracks/responds to changes, to ignore it. I think this has now been replaced by more `specific` flags, such as `skipReconciliation`. So we can certainly revise this flag, and remove/replace it throughout the system.